TGC Comprehensive Rules.md

```diff
@@ -0,0 +1,183 @@
+<h1> Comprehensive TGC Rules </h1>
+
+<h2> 0: About this Document </h2> <br>
+
+0.1: This is the TGC Comprehensive Rules, explaining how to play the TGC. <br>
+0.2: Any deviations between this document, other documents and/or the Wiki then this document\'s wording is the ruling one.<br>
+    0.2.1: An exception is given to the rulings document, which can overrule these rules when talking about specific cards. <br>
+0.3: Sections marked as ".x:" are clarifications; they are not necessary rules text but examples, reminders and etc to make learning the game easier. <br>
+    0.3.1: Sections ".x:" also fall under rule 0.2 in that if there were to be deviations or contradictions between the ".x:" sections and the main sections of this document then the non-".x:" sections take precedence over the ".x:" ones. <br>
+
+<h2> 1: Card & Game Basics </h2> 
+
+1.1: To start a game you need two players. <br>
+
+1.2: Each player needs a deck of cards to play. <br>
+    1.2.1: Decks must include 40 total cards.  <br>
+    1.2.2: You can run up to 4 copies of a specific card in your deck. <br>
+    1.2.3: All cards in your deck must be a legal TGC card. <br>
+    1.2.4: The following rules are deckbuilding rules for the tournament standard of TGC and may be ignored freely in casual play. <br>
+        1.2.4.1: When building your deck choose 3 card subtypes. Every card in your deck must have at least one of these subtypes. <br>
+        1.2.4.2: Every card in your deck must be legal, not banned and not be misprint. <br>
+
+1.3: A coin toss is done to decide who goes first. <br>
+    1.3.1: The coin toss decides who goes first, not who decides to go first. <br>
+    1.3.x: This means a player is decided to be heads and the other tails; whichever side the coin lands on goes first. <br>
+
+1.4: Each player draws 5 cards from their deck. <br>
+1.5: Then, each player may mulligan cards in their hand. <br>
+    1.5.1 During the mulligan each player may discard a card (face-down) from their hand to draw 1. <br>
+        1.5.1.x: The opponent is not allowed to see the discarded cards but can know how many cards you chose to discard.  <br>
+    1.5.2: They can perform this up to 3 times. <br>
+    1.5.3: After both players have finished mulliganing shuffle all discarded cards back into the deck. <br>
+
+1.6: Both players starts with 20 Heathshards (or HS) and 1 Plasma. <br>
```

Comment:
Health?

Reply:
They were already HS on the older version. I agree it's a bit of a silly name but it's gotta stay unless we also change the card edit PR as well, since the cards also mention Healthshards.